src/uct/ib/mlx5/exp/ib_exp_md.c

```diff
@@ -553,6 +553,11 @@ static ucs_status_t uct_ib_mlx5_exp_md_open(struct ibv_device *ibv_device,
         goto err_free;
     }
 
+    status = uct_ib_device_init_ports(dev, ibv_device);
```

Comment:
can we unite uct_ib_device_init_ports with uct_ib_query_device to avoid code duplication?
uct_ib_query_device is called anyway from every md consgructor